src/routes/rss.xml/+server.ts

```diff
@@ -0,0 +1,57 @@
+import { error } from '@sveltejs/kit';
+import type { RequestHandler } from './$types';
+import { getStoryblok } from '$lib/storyblok';
+import type { BlogPostStoryblok } from '$types/bloks';
+
+export const GET: RequestHandler = async () => {
+  const baseUrl = 'https://www.significa.co/blog/';
+  const storyblok = getStoryblok();
+
+  try {
+    const posts: BlogPostStoryblok[] = await storyblok.getAll('cdn/stories', {
+      version: 'published',
+      content_type: 'blog-post',
+      sort_by: 'first_published_at:desc'
+    });
+
+    const { data } = await storyblok.get('cdn/stories/blog');
+    const { seo_title, seo_description } = data.story.content;
+
+    const body = render(baseUrl, seo_title, seo_description, posts);
+
+    const headers = {
+      'Cache-Control': `max-age=0, s-max-age=600`,
+      'Content-Type': 'application/xml'
+    };
+
+    return new Response(body, { headers });
+  } catch (err) {
+    console.error(err);
+    throw error(404, 'Not found');
+  }
```

Comment:
This means that all errors that happen in this code will be shown as 404
For example if Storybklok is down, we have an internet hickup or the most plausible one: the token for SB is wrong - it will return 404. Which is not ideal, a 500 would make much more sense.

The best way is to only throw 404 when the `err` is actually a SB 404.

(500 should happen by default when the error is not handled)